books/free-programming-books-es.md

```diff
@@ -216,13 +216,17 @@
 
 #### AngularJS
 
-> :information_source: Véase también &#8230; [Angular](#angular)
-
 * [AngularJS](https://eladrodriguez.gitbooks.io/angularjs) - Elad Rodriguez (HTML)
+* [Angular 2](https://rldona.gitbooks.io/angular-2-book) - Raúl López (HTML)
+* [Aprendiendo Angular](https://ngchallenges.gitbook.io) - Vanessa Aristizabal (HTML)
```

Comment:
not a book

Reply:
The link was removed because it was a tutorial blog.